parse.go

```diff
@@ -222,10 +222,11 @@ type ParseOption func(*parseOptSet)
 
 // parseOptSet represents the flattened option set after all ParseOptions have been applied.
 type parseOptSet struct {
-	skipMetadataReadOnNewParserInit bool
-	allowMismatchPixelDataLength    bool
-	skipPixelData                   bool
-	skipProcessingPixelDataValue    bool
+	skipMetadataReadOnNewParserInit  bool
+	allowMismatchPixelDataLength     bool
+	skipPixelData                    bool
+	skipProcessingPixelDataValue     bool
+	allowErrorMetaElementGroupLength bool
```

Comment:
nit: can we update this to match AllowMissingMetaElementGroupLength (e.g replace Error with Missing)?

Reply:
Renamed per recommendation